packages/core/src/amazonqGumby/chat/controller/controller.ts

```diff
@@ -338,6 +400,7 @@ export class GumbyController {
         }
 
         const projectPath = transformByQState.getProjectPath()
+        // TODO: remove deprecated metric once BI started using new metrics
```

Comment:
Please add an appropriate test in `core/src/test/techdebt.test.ts`

Reply:
This is super useful! Although at the moment our BI team is still discussing on a date, so this PR will be delayed a bit until I get a committed date.